src/main/scala/subsystem/Configs.scala

```diff
@@ -254,14 +254,19 @@ class WithIncoherentTiles extends Config((site, here, up) => {
   )
 })
 
-class WithRV32 extends Config((site, here, up) => {
-  case XLen => 32
-  case RocketTilesKey => up(RocketTilesKey, site) map { r =>
-    r.copy(core = r.core.copy(
-      fpu = r.core.fpu.map(_.copy(fLen = 32)),
-      mulDiv = Some(MulDivParams(mulUnroll = 8))))
-  }
-})
+class WithRV32 extends Config(
+  new Config((site, here, up) => {
+    case RocketTilesKey => up(RocketTilesKey) map { r =>
+      r.copy(core = r.core.copy(
+        fpu = r.core.fpu.map(_.copy(fLen = 32)),
+        mulDiv = Some(MulDivParams(mulUnroll = 8))
+      ))
+    }
+  }) ++
+    new Config((site, here, up) => {
+      case XLen => 32
+    })
```

Comment:
LGTM, since original CDE behavior was exactly a bug, however my daily CI(playground) didn't figure it out.
I believe this is a necessary bugfix in RC, but need more documentation on the CDE side.

Reply:
It is quite disturbing to me that `WithRV32` is a very different to other `With` configurations. Other `With` configs are only intended to modify cores (`RocketTiles`) that comes before them, but changing `XLen` is a more fundamental thing as many values like `beatBytes` of various buses depend on it. Probably it should be set earlier ...?

Reply:
We believe the recent cde commit introduced a bug in CDE. 

The design of this config is correct, and consistent with the expected behavior of cde.